service/controller/v11/adapter/auto_scaling_group_test.go

```diff
@@ -156,9 +156,9 @@ func TestWorkerCountRatioMaxBatchSize(t *testing.T) {
 			expectedRatio: "1",
 		},
 		{
-			description:   "scaling down to zero worker, none should be up",
+			description:   "scaling down to zero worker, ratio should be one, because it should never be zero",
 			workers:       0,
-			expectedRatio: "0",
+			expectedRatio: "1",
```

Comment:
This is the test that changes because of the one worker update fix. I think having 0 workers should lead to an error on a lot of different places so I am not sure how valid this test is anyway.